predictive_tracking_V2.py

```diff
@@ -0,0 +1,321 @@
+"""
+Predictive Predator Tracker
+Author: Ben McEwen
+
+The class predator tracker takes mp4 file input (standard extract.py format). 
+The centroid of the animal is determined (find_center()) and a Kalman filter is used to smooth the
+movement and reduce noise introduced by occlusion (kalman_filter()). 
+A moving average filter and Kalman filter is used to find the average velocity and this is used to 
+calculate the future position of the animal at a given time (predict_location()).
+"""
+
+
+import cv2
+import numpy as np
+import math
+import matplotlib.pyplot as plt
+import time
+
+from kalman.main import main
+
+class Predator_tracker:
+    """Predator tracker class: - Finds centroid of animal
+                               - Uses kalman filter to smooth effects of occlusion
+                               - Determines direction vector of animal
+                               - Predicts future position of animal"""
+    Green = (0,255,0)
+    Red = (0,0,255)
+    Blue = (255,0,0)
```

Comment:
constant value are normally named in all upper case in Python. Can you rename please?